grails-app/conf/runtime.groovy

```diff
@@ -18,7 +18,7 @@ def dbHost = getInstanceConfig('dbHost') ?: 'localhost',
     dbSchema = getInstanceConfig('dbSchema') ?: appCode,
     dbUser = getInstanceConfig('dbUser') ?: appCode,
     dbPassword = getInstanceConfig('dbPassword'),
-    sslSuffix = dbHost == 'localhost' ? '&useSSL=false' :  ''
+    sslSuffix = dbHost == 'localhost' ? '&useSSL=false&allowPublicKeyRetrieval=true' :  ''
```

Comment:
This bit has been improved in the mysql8 branch, where we consistently spec these driver-specific properties in the config block below (vs. URL query params).  Will merge that now and then can update/merge this PR.